.github/workflows/ci.yml

```diff
@@ -242,10 +242,6 @@ jobs:
     name: Breeze2 tests
     runs-on: ${{ fromJson(needs.build-info.outputs.runsOn) }}
     needs: [build-info]
-    defaults:
-      run:
-        shell: bash
-        working-directory: ./dev/breeze
```

Comment:
And one more thing - > In this case you actually need to also make sure that "dev/breeze" is installed. It used to be doneby working-directory but now I tihnk it's better to keep it consistent with other steps. Few lines below something like that should do - first line will install the dependencies from the right place, and second line will run pytest tests in the righ directory.

```
      - run: pip install ./dev/breeze 
      - run: python3 -m pytest -n auto --color=yes
        working-directory: ./dev/breeze
```

Reply:
(I am not sure if the syntax is correct - but you should get the point ).

Reply:
I checked the error, and it said:
`python3: No module named pytest`, it means that pytest is running in an incorrect path. With these lines:

```
- run: python3 -m pytest -n auto --color=yes 
        working-directory: ./dev/breeze

``` 
I will use pytest in the right path where pytest is installed.

Trying it!